doc/man3/EVP_KDF_set1_pass.pod

```diff
@@ -0,0 +1,78 @@
+=pod
+
+=head1 NAME
+
+EVP_KDF_set1_pass, EVP_KDF_set1_salt, EVP_KDF_set_iter, EVP_KDF_set_md,
+EVP_KDF_set1_key - generic EVP KDF support routines
+
+=head1 SYNOPSIS
+
+ #include <openssl/kdf.h>
+
+ int EVP_KDF_set1_pass(EVP_KDF_CTX *kctx, const unsigned char *pass,
+                       size_t passlen);
+ int EVP_KDF_set1_salt(EVP_KDF_CTX *kctx, const unsigned char *salt,
+                       size_t saltlen);
+ int EVP_KDF_set_iter(EVP_KDF_CTX *kctx, int iter);
+ int EVP_KDF_set_md(EVP_KDF_CTX *kctx, const EVP_MD *md);
+ int EVP_KDF_set1_key(EVP_KDF_CTX *kctx, const unsigned char *key,
+                      size_t keylen);
```

Comment:
You call the context pointer `ctx` in `EVP_KDF_CTX.pod`, but `kctx` here...  any reason why?

Reply:
There is no good reason. The ones with `kctx` (in several pod files) are documenting macros while `EVP_KDF_CTX.pod` is documenting real functions. I will make them more consistent.

Reply:
Done.